docs/man/genericups.txt

```diff
@@ -219,6 +229,11 @@ UPS TYPES
 
      [CP=RTS] [OL=CTS] [LB=-DCD] [SD=DTR]
 
+23 = Generic FTTx (Fiber to the x) battery backup
+     with 4-wire telemetry interface
+
+     [CP=RTS] [OL=CTS] [LB=-DCD] [RB=-RNG] [BYPASS=-DSR] [SD=none]
```

Comment:
@dogtopus : In the patch description you mentioned a "NULL" vs. "none" - is this `SD=none` one of such cases? I did not find an earlier particular handling for "none" string in `genericups.c` so got no strong opinion... just wondering if that `strstr` call checking for "NULL" string should also check for "or `none`" and that's it?

Reply:
@jimklimov Yes, that's exactly what I meant. Documentations have none as the placeholder for not applicable/disabled signals but I used NULL instead (mostly because I implemented the feature before I actually read that part of the doc).

Reply:
Followed up with https://github.com/dogtopus/nut/pull/1

Wondering now if an `SD=none` as documented with the original PR would have been in fact handled when parsing the `upstype` from `ups.conf` as documented? Just in case, did you check that ability to customize?